README.md

```diff
@@ -51,6 +51,20 @@ Feel free to open issues and/or fork the project to contribute ([see Development
 
 Chrome 48+, Firefox 44+, Safari 8+, IE 10+.
 
+## Semantic Versioning
+
+The dress-code library follows [semantic version standards](https://semver.org/).
```

Comment:
This is a nitpick but we should try to use a common way to refer to the library. Most of the times we refer to it as "The Dress Code" so I guess either that or another wording should be the only one used everywhere

Reply:
I agree